### PR TITLE
Add a switch to show transliteration of sundar gutka bani names

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Fix issue #1639 
sttm-sg

### CODE REVIEW

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx [hunk 1]
```diff
@@ -83,8 +84,27 @@ const SundarGutka = ({ isShowTranslitSwitch = false, onScreenClose }) => {
...
+                  <div className="switch xs-small">
+                    <input
+                      id="translate-eng"
+                      type="checkbox"
+                      checked={isEngTransliterated}
+                      onChange={() => {
+                        const newState = !isEngTransliterated;
+                        setEngTransliterate(newState);
+                        if (isEngTransliterated) {
+                          onToggle(newState);
```
Comment (theamanjs) [inline www/main/addons/sundar-gutka/components/SundarGutka.jsx:100]: onToggle is not defined

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx [hunk 2]
```diff
@@ -116,11 +138,19 @@ const SundarGutka = ({ isShowTranslitSwitch = false, onScreenClose }) => {
...
           <div className={`bani-extras overlay-ui ${overlayClassName}`}>
             {nitnemBanis.length > 0 && (
-              <ExtraBani title="Nitnem Banis" banis={nitnemBanis} loadBani={loadBani} />
-            )}
+              <ExtraBani
+                title="Nitnem Banis"
+                banis={nitnemBanis}
+                loadBani={loadBani}
+                isEngTransliterated={isEngTransliterated}
+              />            )}
```
Comment (theamanjs) [inline www/main/addons/sundar-gutka/components/SundarGutka.jsx:146]: looks like indentation issue

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx [hunk 3]
```diff
@@ -116,11 +138,19 @@ const SundarGutka = ({ isShowTranslitSwitch = false, onScreenClose }) => {
...
             {popularBanis.length > 0 && (
-              <ExtraBani title="Popular Banis" banis={popularBanis} loadBani={loadBani} />
-            )}
+              <ExtraBani
+                title="Popular Banis"
+                banis={popularBanis}
+                loadBani={loadBani}
+                isEngTransliterated={isEngTransliterated}
+              />            )}
```
Comment (theamanjs) [inline www/main/addons/sundar-gutka/components/SundarGutka.jsx:153]: indentation issue here as well

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx (hunk 1 above)
Comment (Gauravjeetsingh) [reply www/main/addons/sundar-gutka/components/SundarGutka.jsx]: Removed, that code wasn't needed anymore.

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx (hunk 2 above)
Comment (Gauravjeetsingh) [reply www/main/addons/sundar-gutka/components/SundarGutka.jsx]: Fixed

File: www/main/addons/sundar-gutka/components/SundarGutka.jsx (hunk 3 above)
Comment (Gauravjeetsingh) [reply www/main/addons/sundar-gutka/components/SundarGutka.jsx]: Fixed

**Pull request merged**